op_map.cc

```diff
@@ -970,6 +970,19 @@ struct L2NormalizationMapper
 };
 
 struct ReshapeMapper : public OpMapperBase<TfLiteReshapeParams> {
+  virtual bool IsOpSupported(TfLiteContext* context,
+                             TfLiteNode* node,
+                             const TfLiteRegistration* registration) const {
+    auto input_tensor = context->tensors[node->inputs->data[0]];
+    auto output_index = node->outputs->data[0];
+
+    if (context->tensors[output_index].dims->size == 0) {
+      TFLITE_LOG_PROD(TFLITE_LOG_ERROR,
```

Comment:
use warning instead?

Reply:
also changed error to warning/info level for delegete_main.c raised by zero-sized tensor.

Reply:
done